src/com/untamedears/JukeAlert/util/Utility.java

```diff
@@ -154,15 +160,30 @@ public static Snitch findClosestOwnedSnitch(Player player) {
             }
         }
         return closestSnitch;
+    } */
+    
+    public static Snitch findClosestSnitch(Location loc) {
```

Comment:
The biggest issue with this is it could be used to discover enemy snitches.

For instance, consider this layout:

`
   s
+----+
      |e
   p  |
`

Let's assume p is player, lines are walls, s is the snitch I know about, e is an enemy snitch. 

E is closest to me; so, this method returns e. However, I didn't know about e, I only knew about s. Now, I have a pretty good idea (if I'm paying attention) that someone else has a snitch nearby that shouldn't be there.

I'd recommend altering this function or having another version of it that pulls the membership check within same as prior.